core/sup_tree_core/executor_pool/async_job_log_writer.ex

```diff
@@ -0,0 +1,46 @@
+# Copyright(c) 2015-2019 ACCESS CO., LTD. All rights reserved.
+
+use Croma
+
+defmodule AntikytheraCore.ExecutorPool.AsyncJobLog.Writer do
+  @moduledoc """
+  A `GenServer` for logging, which is used in `AntikytheraCore.ExecutorPool.AsyncJobRunner`.
+  """
+
+  use GenServer
+  alias Antikythera.{Time, ContextId}
+  alias AntikytheraCore.GearLog.LogRotation
+
+  @rotate_interval 24 * 60 * 60 * 1000
+
+  def start_link([]) do
+    GenServer.start_link(__MODULE__, [], [name: __MODULE__])
+  end
+
+  @impl true
+  def init([]) do
+    log_file_path = AntikytheraCore.Path.core_log_file_path("async_job")
+    state = LogRotation.init(@rotate_interval, log_file_path, write_to_terminal: false)
+    {:ok, state}
+  end
+
+  @impl true
+  def handle_cast(message, state) do
+    log = {Time.now(), :info, ContextId.system_context(), message}
+    {:noreply, LogRotation.write_log(state, log)}
+  end
+
+  @impl true
+  def handle_info(:rotate, state) do
+    {:noreply, LogRotation.rotate(state)}
+  end
+
+  @impl true
+  def terminate(_reason, state) do
+    LogRotation.terminate(state)
+  end
+
```

Comment:
Please add the following comments in case you implemented public API of GenServer.
(Please refer [here](https://github.com/access-company/antikythera/blob/master/core/sup_tree_core/cluster_hosts_poller.ex#L72).)
```
#
# Public API
#
```

Reply:
I fixed it.
https://github.com/access-company/antikythera/pull/93/commits/796fed88a2f7dc08386b8b0b225a1c79cba24e50